Metrics collision with internal metrics

Hi! I'm getting the same error whenever I download a prebuilt executable or run in via docker image:

When I open http://localhost:9108/metrics
```
An error has occurred while serving metrics:

6 error(s) occurred:
* collected metric process_cpu_seconds_total counter:<value:0.2 >  has help "Total user and system CPU time spent in seconds." but should have "Graphite metric process_cpu_seconds_total"
* collected metric process_virtual_memory_bytes gauge:<value:7.31475968e+08 >  has help "Virtual memory size in bytes." but should have "Graphite metric process_virtual_memory_bytes"
* collected metric process_resident_memory_bytes gauge:<value:1.3062144e+07 >  has help "Resident memory size in bytes." but should have "Graphite metric process_resident_memory_bytes"
* collected metric process_start_time_seconds gauge:<value:1.63203198914e+09 >  has help "Start time of the process since unix epoch in seconds." but should have "Graphite metric process_start_time_seconds"
* collected metric process_open_fds gauge:<value:12 >  has help "Number of open file descriptors." but should have "Graphite metric process_open_fds"
* collected metric process_max_fds gauge:<value:1.048576e+06 >  has help "Maximum number of open file descriptors." but should have "Graphite metric process_max_fds"
```

And I'm also unable to build it from source:

```
WARN [runner] Can't run linter goanalysis_metalinter: buildir: failed to load package tsdb: could not load export data: no export data for "github.com/prometheus/prometheus/tsdb" 
ERRO Running error: buildir: failed to load package tsdb: could not load export data: no export data for "github.com/prometheus/prometheus/tsdb" 
make: *** [Makefile.common:192: common-lint] Error 3
```

Thank you!

--- a/README.md
+++ b/README.md
@@ -166,7 +166,8 @@ In the transition to the Graphite data model and back, information is lost.
 Additionally, default metrics conflict between the client libraries and the exporter.
 
 Instead, configure Prometheus to scrape your application directly, without the exporter in the middle.
-If you absolutely must push, consider the [Grafana agent](https://github.com/grafana/agent) instead.
+For batch or ephemeral jobs, use the [pushgateway](https://prometheus.io/docs/practices/pushing/) [integration](https://github.com/prometheus/client_python#exporting-to-a-pushgateway).
+If you absolutely must push, consider [PushProx](https://github.com/prometheus-community/PushProx) or the [Grafana agent](https://github.com/grafana/agent) instead.
 
 
 [circleci]: https://circleci.com/gh/prometheus/graphite_exporter

--- a/README.md
+++ b/README.md
@@ -149,9 +149,7 @@ docker run -d -p 9108:9108 -p 9109:9109 -p 9109:9109/udp \
         prom/graphite-exporter --graphite.mapping-config=/tmp/graphite_mapping.conf
 ```
 
-## Importing Whisper data
-
-**EXPERIMENTAL**
+## **Experimental**: Importing Whisper data
 
 Import data from Graphite using the bundled `getool`.
 See `getool create-blocks --help` for usage.
@@ -160,6 +158,16 @@ To import long-term data in a reasonable amount of resources, increase the durat
 The `--block-duration` must be a power of two in hours, e.g. `4h`, `8h`, and so on.
 
 To merge the data into an existing Prometheus storage directory, start Prometheus with the `--storage.tsdb.allow-overlapping-blocks` flag.
+
+## Incompatibility with Graphite bridge
+
+This exporter does not work in combination with the [Java client](https://prometheus.github.io/client_java/io/prometheus/client/bridge/Graphite.html) or [Python client](https://github.com/prometheus/client_python#graphite) Graphite bridge.
+In the transition to the Graphite data model and back, information is lost.
+Additionally, default metrics conflict between the client libraries and the exporter.
+
+Instead, configure Prometheus to scrape your application directly, without the exporter in the middle.
+If you absolutely must push, consider the [Grafana agent](https://github.com/grafana/agent) instead.
+
 
 [circleci]: https://circleci.com/gh/prometheus/graphite_exporter
 [hub]: https://hub.docker.com/r/prom/graphite-exporter/

